packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -234,6 +234,10 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
     final TestWidgetInspectorService service = TestWidgetInspectorService();
     WidgetInspectorService.instance = service;
 
+    tearDown(() {
+      service.resetAllState();
```

Comment:
Could we avoid creating special API for this by just always instantiating a new TestWidgetInspectorService for each test run, e.g. in `setUp()`?